examples/ae_examples/cvae_dim_example/mnist_model.py

```diff
@@ -0,0 +1,15 @@
+import torch
```

Comment:
We've been putting in our models for examples into the `examples/models` folder in case they might be reused. I think there might be an MnistNet model that accepts an "input space" size which would have the same implications as `latent_dim` here.

Reply:
I guess there actually isn't one, but I think Yuchong is introducing one that has a variable input size as well. So perhaps we can share the model structure for this example and his.

Reply:
Sure, this model and Yuchong's model are used for the same thing, so it totally makes sense for these examples to share the same model. I am going to use Yuchong's model then. On another note, just noticed we have 2 duplicate models in our `examples/models`` directory (mlp_classifier and logistic_regression), not sure if this was intentional.

Reply:
Ah, if there are duplicates, please feel free to remove/merge them 🙂